config/intellij-idea-inspections.xml

```diff
@@ -3768,6 +3774,9 @@
     <option name="ignoreEmptySuperMethods" value="false"/>
     <option name="onlyReportWhenAnnotated" value="true"/>
   </inspection_tool>
+  <!-- temporal suppression until issues are resolved -->
```

Comment:
Can you tie this to an issue number if this is going to be merged as is?

Reply:
I need to create separate issue, as #11566 is already overused for numerous updates

Reply:
done